### ISSUE TITLE
[Help]: Need a list of hardware configurations.

### ISSUE THREAD

**Issue by isKEKE** [open-mmlab/Amphion#151]
Hello Team

What is the minimum GPU memory to train TTA? I tried to train with 4060TI but it says not enough memory.

Is it possible to provide a detailed list of hardware configuration?

Thanks

**Comment by HeCheng0625** [comment]
Hi, we use one 24G 4090 GPU to train the TTA model, you can decrease the batch size (you may need decrease the learning rate at the same time) if you don't have enough RAM.

**Comment by isKEKE** [comment]
Thanks for the advice.
